plugins/web/opentelemetry-plugin-user-interaction/src/userInteraction.ts

```diff
@@ -193,11 +194,36 @@ export class UserInteractionPlugin extends BasePlugin<unknown> {
             return listener.apply(target, args);
           }
         };
+        plugin._wrappedListeners.set(listener, patchedListener);
         return original.call(this, type, patchedListener, useCapture);
       };
     };
   }
 
+  /**
+   * This patches the removeEventListener of HTMLElement to handle the fact that
+   * we patched the original callbacks
+   * This is done when zone is not available
+   */
+  private _patchRemoveEventListener() {
+    const plugin = this;
+    return (original: Function) => {
+      return function removeEventListenerPatched(
+        this: HTMLElement,
+        type: any,
+        listener: any,
+        useCapture: any
+      ) {
+        const wrappedListener = plugin._wrappedListeners.get(listener);
```

Comment:
I would also remove the wrapped listener from the weakmap here

Reply:
Good call; patching now.